.circleci/config.yml

```
@@ -94,113 +54,29 @@ jobs:
           paths:
             - './*.tgz'
 
-  airflow-local-on-k8s-1-14:
+  airflow-test:
     machine:
       image: ubuntu-1604:201903-01
+    parameters:
+      executor:
+        type: string
+        default: "LocalExecutor"
+      kube_version:
+        type: string
+        default: "1.18.2"
     environment:
       KUBE_VERSION: 1.14.10
       EXECUTOR: "LocalExecutor"
```

We can remove this now I think

---

oops, good catch

---

or use parameters here:

```
```suggestion
      KUBE_VERSION: << parameters.kube_version >>
      EXECUTOR: << parameters.executor >>
```

---

idk if that would work or not though